### PR TITLE
Issue #18: Cleaned up verbose output.

### PR DESCRIPTION
**Pull request by jcherry-risklens**
See issue #18. 

Cleaned up several verbose outputs to only display when verbose=True, including TQDM bars.

### CODE REVIEW

File: verispy/veris.py [hunk 1]
```diff
@@ -368,7 +370,7 @@ def json_to_df(self, filenames=None, keep_raw=False, schema_path=None, schema_ur
         """
 
         # load schema
-        if verbose: print('Loading schema')
+        if verbose : print('Loading schema')
```
Comment (tbyers-risklens) [inline verispy/veris.py]: PEP 8 says to leave this how I did it

https://www.python.org/dev/peps/pep-0008/#id26

File: verispy/veris.py [hunk 2]
```diff
@@ -168,8 +169,9 @@ def var_amt_enum_checker(dfitem, enumitem, variety_or_amt):
 
         comb_df = pd.DataFrame()
         top_level_enums = [enum for enum in enums]
-        print('Building enumeration columns.')
-        for col in tqdm(raw_df.columns):
+        if verbose : print('Building enumeration columns.')
```
Comment (tbyers-risklens) [inline verispy/veris.py]: Yes: `if verbose: print`
No: `if verbose : print`

https://www.python.org/dev/peps/pep-0008/#id26

File: verispy/veris.py [hunk 3]
```diff
@@ -35,9 +35,9 @@ def __init__(self, json_dir=None, verbose=False, schema_url=veris_const.SCHEMA_U
         self.json_dir = json_dir
         if json_dir:  # build when building data frame
             self.filenames = glob.glob(os.path.join(json_dir, '*.json'))
-            print('Found {} json files.'.format(len(self.filenames)))
+            if verbose : print('Found {} json files.'.format(len(self.filenames)))
```
Comment (tbyers-risklens) [inline verispy/veris.py]: Yes: if verbose: print
No: if verbose : print

https://www.python.org/dev/peps/pep-0008/#id26

File: verispy/veris.py [hunk 4]
```diff
@@ -67,9 +67,10 @@ def _rawjson_to_df(self, filenames, verbose=False):
         pd DataFrame 
             The raw data (no enuemerations)
         """
-        print('Loading JSON files to DataFrame.')
+        if verbose : print('Loading JSON files to DataFrame.')
         jsons = []
-        for j in tqdm(filenames):
+        t = tqdm(filenames) if verbose else filenames
+        for j in t:
```
[the remark this answers is not in the document]
Comment (jcherry-risklens) [reply verispy/veris.py]: Yeah, TQDM often needs a few tricks to get it to work in production code. That's the approach I usually use for it, though I'm sure there's others.

**Pull request merged**